doc/source/rllib/rllib-algorithms.rst

```diff
@@ -7,236 +7,176 @@
-`Fully Independent Learning`_    Depends on bootstrapped algorithm
--------------------------------- ---------------------------------------------------------------------------------------
-`Shared Critic Methods`_         Depends on bootstrapped algorithm
-================================ =======================================================================================
-
-.. _`+autoreg`: rllib-models.html#autoregressive-action-distributions
-.. _`+LSTM auto-wrapping`: rllib-models.html#built-in-models
-.. _`+parametric`: rllib-models.html#variable-length-parametric-action-spaces
-.. _`Rainbow`: rllib-algorithms.html#dqn
-.. _`+RNN`: rllib-models.html#rnns
-.. _`+Attention`: rllib-models.html#attention
-.. _`TS`: rllib-algorithms.html#lints
-
-Offline
-~~~~~~~
-
-.. _bc:
-
-Behavior Cloning (BC; derived from MARWIL implementation)
----------------------------------------------------------
-|pytorch| |tensorflow|
-`[paper] <http://papers.nips.cc/paper/7866-exponentially-weighted-imitation-learning-for-batched-historical-data>`__
-`[implementation] <https://github.com/ray-project/ray/blob/master/rllib/algorithms/bc/bc.py>`__
-
-Our behavioral cloning implementation is directly derived from our `MARWIL`_ implementation,
-with the only difference being the ``beta`` parameter force-set to 0.0. This makes
-BC try to match the behavior policy, which generated the offline data, disregarding any resulting rewards.
-BC requires the `offline datasets API <rllib-offline.html>`__ to be used.
-
-Tuned examples: `CartPole-v1 <https://github.com/ray-project/ray/blob/master/rllib/tuned_examples/bc/cartpole-bc.yaml>`__
-
-**BC-specific configs** (see also `common configs <rllib-training.html#common-parameters>`__):
-
-.. autoclass:: ray.rllib.algorithms.bc.bc.BCConfig
-   :members: training
+Overview
+~~~~~~~~
+
++-----------------------------------------------------------------------------+------------------------------+------------------------------------+--------------------------------+
+| **Algorithm**                                                               | **Single- and Multi-agent**  | **Multi-GPU (multi-node)**         | **Action Spaces**              |
++-----------------------------------------------------------------------------+------------------------------+------------------------------------+--------------------------------+
+| **On-Policy**                                                                                                                                                                    |
++-----------------------------------------------------------------------------+------------------------------+------------------------------------+--------------------------------+
+| :ref:`PPO (Proximal Policy Optimization) <ppo>`                             | |single_agent| |multi_agent| | |multi_gpu| |multi_node_multi_gpu| | |cont_actions| |discr_actions| |
```

Comment:
Make more clear, what is the difference between multi-gpu and multi-node-multi-gpu.

Reply:
Good point!

Reply:
Done.